ha/alert/alert_monitor.py

```diff
@@ -37,17 +39,17 @@ def __init__(self):
 
         # get environment variables
         self.crm_env = self._get_env()
-        # Modules like Node, Resource, Fencing
-        self.alert_event_module = None
-        # Module event like node became member or node lost
-        self.alert_event_type = None
+        _alert_event_filter = AlertEventFilter()
+        _alert_event_filter.initialize_crm(self.crm_env)
```

Comment:
This function is redundant. Please remove 
You are passing self.crm_env and assigning it to the same variable  self.crm_env inside the function.